api/app/reviews/api.py

```diff
@@ -90,21 +94,41 @@
     'scores': fields.List(fields.Nested(review_scores_fields), attribute='review_scores')
 }
 
+reference_fields = {
+    'firstname': fields.String,
+    'title': fields.String,
+    'lastname': fields.String,
+    'relation': fields.String,
+    'uploaded_document': fields.String,
+}
+
 review_fields = {
     'review_form': fields.Nested(review_form_fields),
     'response': fields.Nested(response_fields),
     'user': fields.Nested(user_fields),
     'reviews_remaining_count': fields.Integer,
-    'review_response': fields.Nested(review_response_fields)
+    'review_response': fields.Nested(review_response_fields),
+    'references': fields.List(fields.Nested(reference_fields)),
 }
 
+
 class ReviewResponseUser():
-    def __init__(self, review_form, response, reviews_remaining_count, review_response=None):
+    def __init__(self, review_form, response, reviews_remaining_count, reference_response=None):

```

Comment:
Nit: this parameter name is singular, but you're actually expecting a list, so would be better to name it `reference_responses` (plural)